src/Evolution/Systems/Cce/InitializeCce.hpp

```diff
@@ -32,19 +88,31 @@ namespace Cce {
  * B &= - \frac{1}{2} R^3 (J|_{r = R} + R \partial_r J|_{r = R})
  * \f}
  */
-template <template <typename> class BoundaryPrefix>
-struct InitializeJ {
-  using boundary_tags = tmpl::list<BoundaryPrefix<Tags::BondiJ>,
-                                   BoundaryPrefix<Tags::Dr<Tags::BondiJ>>,
-                                   BoundaryPrefix<Tags::BondiR>>;
+struct InitializeJInverseCubic : public InitializeJ {
```

Comment:
Name `InverseCubic` in the input file, either by renaming the class or by adding a `static std::string name()` function, so that it reads:
```yaml
InitializeJ:
  InverseCubic:
```